src/cli.rs

```diff
@@ -49,6 +49,11 @@ pub struct Opts {
     /// Options to modify logging and error-handling behavior.
     #[command(flatten)]
     pub logging: LoggingOpts,
+
+    /// By default, ghciwatch will interrupt reloads if a file changes. If you want ghciwatch to
+    /// avoid interrupting reloads, set this flag.
+    #[arg(long = "no-interrupt-reloads")]
+    pub no_interrupt_reloads: bool,
```

Comment:
I'm not super happy with a `--no-interrupt-reloads` flag - feels kind of awkward. But `--interrupt-reloads` is the default behavior.

We could switch the default back to `no-interrupt`, and then this flag can be `--interrupt-reloads`.

Made a slack convo to ask about this: 
https://mercurytechnologies.slack.com/archives/CUJ89R895/p1702483630925389

Reply:
There's an open issue for automatic negation flags (i.e. adding `--interrupt-reloads` and `--no-interrupt-reloads` at once) here:

https://github.com/clap-rs/clap/issues/815